client/tinychain/ml/layers.py

```diff
@@ -0,0 +1,65 @@
+from typing import List
+
+from tinychain.collection.tensor import einsum, Dense, Tensor
+from tinychain.ml import Layer, Sigmoid, DiffedParameter, Parameter
+from tinychain.ref import After
+
+
+class ConvLayer(Layer):
```

Comment:
please add a `TODO` here with info about how this relates to the existing `tc.ml.cnn` and `tc.ml.dnn` modules